pom.xml

```diff
@@ -223,9 +240,9 @@
         </pluginManagement>
     </build>
     <properties>
-        <camel.version>3.14.6</camel.version>
-        <spring-boot.version>2.6.11</spring-boot.version>
-        <log4j.version>2.19.0</log4j.version>
+        <camel.version>3.14.7</camel.version>
```

Comment:
3.20.4 is available as well but I am not sure about the compatibility guarantees.